app/controllers/discovered_hosts_controller.rb

```diff
@@ -60,7 +60,7 @@ def destroy
 
   def edit
     Host.transaction do
-      @host = ::ForemanDiscovery::HostConverter.to_managed(@host) unless @host.nil?
+      @host = ::ForemanDiscovery::HostConverter.to_managed(@host, true, false, false) unless @host.nil?
```

Comment:
is this the only place we shouldn't delete the attributes?
we have `to_managed` in `perform_auto_provision` in `discovered_extensions.rb`, if the auto_provision fails what happens?